migrations/postgres/20240603102429_access-tokens_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE access_tokens(
+    id uuid NOT NULL PRIMARY KEY,
+    token_name VARCHAR(100) NOT NULL,
+    token_hash BYTEA NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT now(),
```

Comment:
I would avoid defaults like that, it would be hard to stub this time in tests.

Reply:
Removed